cmd/stack/deploy.go

```diff
@@ -47,6 +50,30 @@ func Deploy(ctx context.Context) *cobra.Command {
 				return err
 			}
 
+			if okteto.IsOkteto() {
+				c, err := okteto.NewOktetoClient()
+				if err != nil {
+					return err
+				}
+				hasAccess, err := utils.HasAccessToNamespace(ctx, s.Namespace, c)
```

Comment:
If that happen when the user tries to create the namespace the backend will return an error when calling create namespace

Reply:
I know, but maybe we can handle that scenario in a different way. You are asking the user if they want to create a namespace, they say yes, and then, it fails because there is a namespace with the same name. Anyways, I think this could be good enough and we can add that case if needed in the future